src/Native/Runtime/gcenv.h

```diff
@@ -40,6 +40,14 @@
     #define FireEtwPinPlugAtGCTime(PlugStart, PlugEnd, GapBeforeSize, ClrInstanceID) 0
     #define FireEtwGCTriggered(Reason, ClrInstanceID) 0
 
+    #ifndef _INC_WINDOWS
```

Comment:
gcenv.h is (will be) shared with the PN build, which this branch of the #ifdef CORERT is specific to.  These typedefs still get used by etwevents.h / eventtrace.h (not yet in CoreRT), which have not yet been modified to remove the uses of these typedefs.   This whole branch of the #ifdef CORERT is a TODO for PN.